proto/pkg/cadvisor/api/cadvisor.proto

```diff
@@ -0,0 +1,171 @@
+syntax = "proto3";
+
+package cadvisor.api;
+option go_package = "./api";
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+
+service ContainerInfoService {
```

Comment:
Do we need this service? Also, why is the rpc named `NodeCapacityStatus`?

Reply:
Unintended commit. Removed.